libraries/AP_DDS/AP_DDS_Topic_Table.h

```diff
@@ -16,81 +16,118 @@ const struct AP_DDS_Client::Topic_table AP_DDS_Client::topics[] = {
     {
         .topic_id = 0x01,
         .pub_id = 0x01,
+        .sub_id = 0x01,
         .dw_id = uxrObjectId{.id=0x01, .type=UXR_DATAWRITER_ID},
+        .dr_id = uxrObjectId{.id=0x01, .type=UXR_DATAREADER_ID},
         .topic_profile_label = "time__t",
         .dw_profile_label = "time__dw",
+        .dr_profile_label = "time__dr",
```

Comment:
Change to `""` because there is no data reader for time. Just Joy

Reply:
Done.